packages/webpack/mixins/esbuild/mixin.core.js

```diff
@@ -0,0 +1,56 @@
+const { Mixin } = require('hops-mixin');
+
+class HopsEsbuildMixin extends Mixin {
+  configureBuild(webpackConfig, { jsLoaderConfig }, target) {
+    const { experimentalEsbuild } = this.options;
+
+    if (!experimentalEsbuild) {
+      return;
+    }
+
+    // NOTE: We don't want to ship esbuild, so we just assume it is installed
+    // when someone wants to use it.
+    // eslint-disable-next-line node/no-missing-require, node/no-extraneous-require
+    const { ESBuildPlugin, ESBuildMinifyPlugin } = require('esbuild-loader');
```

Comment:
The [`ESBuildPlugin` can be removed now.](https://github.com/privatenumber/esbuild-loader/releases/tag/v2.10.0)

Reply:
Hm, but we would still need to boilerplate code for the webpack API adapter, right? Why is it even deprecated?